setup.cfg

```diff
@@ -7,4 +7,5 @@ exclude = mypy/codec/*,mypy/test/data/lib-stub/*,mypy/test/data/fixtures/*
 #   F401: unused identifiers (useless, as it doesn't see inside # type: comments)
 #   W601: has_key() deprecated (false positives)
 #   E701: multiple statements on one line (colon) (we use this for classes with empty body)
-ignore = E251,E128,F401,W601,E701
+#   F403: 'from foo import *' used; unable to detect undefined names (mypy checks this better)
```

Comment:
That rule is there because flake8 is too dumb to figure out how to handle undefined names in this case, not because glob imports are fundamentally wrong.

Easily abused? Sure. Easily replaceable? Not always.